guides/common/modules/proc_adding-an-scc-account.adoc

```diff
@@ -20,23 +20,23 @@ Add the GPG public key for SLES 15 SP3 from https://www.suse.com/support/securit
 By default, it is set to `\https://scc.suse.com`.
 . Optional: Set a *Sync interval* to periodically update the SCC authentication tokens.
 Note that this does not refer to synchronizing content to {Project}.
-. Optional: In the *Use GPG key for SUSE products* field, you can select the previously created content credential to automatically add a GPG public key to your SUSE products.
+. Optional: In the *Use GPG key for SUSE products* field, you can select the previously created content credential to automatically add a GPG public key to your SUSE Products.
```

Comment:
Stating the obvious: product/Product is capitalized inconsistently on this line. I don't have a VM with the right plug-in available so I can't check the context but it does look like "Product" is the right choice here too. Perhaps we could change the label of that field in the web UI? I suppose the PR should be simple, except that I can't find the right place in the code.